src/kernels/ipywidgets/types.ts

```diff
@@ -80,16 +80,10 @@ export interface IWidgetScriptSourceProviderFactory {
     ): IWidgetScriptSourceProvider[];
 }
 
-export const ILocalResourceUriConverter = Symbol('ILocalResourceUriConverter');
-
 /**
  * Given a local resource this will convert the Uri into a form such that it can be used in a WebView.
  */
 export interface ILocalResourceUriConverter {
```

Comment:
Would this still be needed?

Reply:
Yes, we don't want to hardcode the class everywhere, instead just use the interafce.